MAP/src/components/Totals/Totals.js

```diff
@@ -4,7 +4,7 @@ import { fetchTotals } from '../../services';
 import './Totals.css';
 
 const separateNumber = (number) => {
-  const nfObject = new Intl.NumberFormat('en-US');
+  const nfObject = new Intl.NumberFormat('es-ES');
```

Comment:
I wonder, is it possible to get this encoding value preference from the user's browser info? maybe something to consider in the future, but for now this works to achieve the request :+1: